resources/sql/queries.sql

```diff
@@ -33,11 +33,22 @@ INSERT INTO catalogue_item
 (title, formid, resid, wfid)
 VALUES (:title, :form, :resid, :wfid)
 
+-- :name get-resources :? :*
+SELECT
+  id,
+  modifieruserid,
+  resid,
+  start,
+  endt
+FROM resource
+
 -- :name create-resource! :insert
 -- :doc Create a single resource
 INSERT INTO resource
 (id, resid, prefix, modifieruserid)
-VALUES (:id, :resid, :prefix, :modifieruserid)
+VALUES (
+  /*~ (if (:id params) */ :id /*~ */ DEFAULT /*~ ) ~*/,
```

Comment:
Should `id` parameter be removed from here?

Reply:
done